bin/node/runtime/src/lib.rs

```diff
@@ -0,0 +1,353 @@
+//! The Substrate Node Template runtime. This can be compiled with `#[no_std]`, ready for Wasm.
```

Comment:
Could we extend `polkadot/kusama` runtime instead? Perhaps use common stuff from their repo?

One thing that might not work is that we will end up in git dependency hell if we don't solve #20 first.

Reply:
Is there anything in particular that you'd like to get out from their runtime?

Reply:
Talked about this off Github. What we'll do is keep the template for now, and once we have made more substantial progress with the runtimes we'll add a Kusama/Polkadot node for testing.